docs/user-guide/nextcloud-integration/README.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_navigation:
+  title: Nextcloud integration
+  priority: 600
+description: Nextcloud integration
+keywords: integrations, apps, Nextcloud
+
+---
+
+# Nextcloud integration
+
+[Nextcloud](https://nextcloud.com/), the worlds’ most deployed on-premises content collaboration platform, and [OpenProject](https://www.openproject.org/), the leading free and open source project management software, join forces. For both Nextcloud and OpenProject **data sovereignty** and **open source** are an integral part of their business strategy and thus the foundation for their alliance.
+
+OpenProject and Nextcloud together combine project management and content collaboration and complement each other’s services. The collaboration of these two companies offers users a new dimension of data sovereignty and increase in productivity. Users will not risk compliance and data leaks by losing control over their content and can now even easily combine both systems.
```

Comment:
- I would talk of "communities" instead of "companies"
- I wonder if the focus on compliance and data leaks is the right focus. I always thought the focus was to solve the two questions: "Where are the files I need to for completing the task?" and "Where shall I store the files when I have completed my taks?"

Reply:
Hi @wielinde,
This text is mainly extracted from a blog post: https://www.openproject.org/blog/openproject-nextcloud-collaboration/ but yes I agree we can focalise more into this two questions.

Reply:
This part has been changed since :)